features/step_definitions/search_steps.rb

```diff
@@ -0,0 +1,35 @@
+When('I type {string} into the search field') do |search_term|
+  @homepage.type_in_search_field(search_term)
+end
+
+When('I click the search button') do
+  @homepage.click_search_button
+end
+
+Then('I should be redirected to the search results page') do
+  expect(page.current_url).to include('/search')
+end
+
+Then('I should see a list of results related to {string}') do |search_term|
+  expect(page).to have_content(search_term)
+end
```

Comment:
what if this method will find similar text data that we put into the search field instead of expected list with items?